### PR TITLE
Refuel change documentation header for API fnc

### PR DESCRIPTION
**Pull request by GitHawk**
Cleanup for #3404

### CODE REVIEW

**Comment by GitHawk** [comment]
This PR is for refuel @esteldunedain

**Comment by jonpas** [comment]
Thank you @GitHawk

File: addons/refuel/functions/fnc_connectNozzleAction.sqf [hunk 1]
```diff
@@ -60,7 +60,7 @@ while {(_closeInMax - _closeInMin) > 0.01} do {
 _closeInDistance = (_closeInMax + _closeInMin) / 2;
 
 //Checks (too close to center or can't attach)
-if (((_startDistanceFromCenter - _closeInDistance) < 0.1) || {!([_target, _unit, _itemClassname] call FUNC(canAttach))}) exitWith {
+if ((_startDistanceFromCenter - _closeInDistance) < 0.1) exitWith {
```
Comment (esteldunedain) [inline addons/refuel/functions/fnc_connectNozzleAction.sqf]: why was this changed? Should it be reverted @GitHawk?

Comment (GitHawk) [inline addons/refuel/functions/fnc_connectNozzleAction.sqf]: Because there is no `ace_refuel_fnc_attach` function.
Since this code was copied from `ace_attach` it was a remnant and has never thrown an error.
In `ace_attach` the` fnc_canAttach` limits the amount of attached objects.

Comment (esteldunedain) [inline addons/refuel/functions/fnc_connectNozzleAction.sqf]: k, np then
